INSTALL.md

```diff
@@ -26,12 +26,15 @@ There are several other distributions of the PureScript compiler available, whic
 * NPM: `npm install -g purescript`
 * Homebrew (for macOS): `brew install purescript`
 * [PSVM](https://github.com/ThomasCrevoisier/psvm-js): `npm install -g psvm`
+* [Hackage](https://hackage.haskell.org/package/purescript): `cabal install purescript` or `stack install purescript`
 
 ## Compiling from source
 
 The easiest way is to use stack:
 
 ```
+$ git clone https://github.com/purescript/purescript.git purescript_compiler
+$ cd purescript_compiler
```

Comment:
`stack unpack` is the thing that downloads the sources here, no need to also clone the repo.